packages/derived-wallet-solana/src/signAptosTransaction.ts

```diff
@@ -58,10 +59,13 @@ export async function signAptosTransactionWithSolana(input: SignAptosTransaction
     serializer.serializeOption(input.rawTransaction.feePayerAddress);
     const authenticator = serializer.toUint8Array();
 
+    const accountIdentity = `${solanaPublicKey.toBase58()}:${domain}`
+    
     return new AccountAuthenticatorAbstraction(
       authenticationFunction,
       signingMessageDigest,
       authenticator,
+      new TextEncoder().encode(accountIdentity)
```

Comment:
i think this is already being computed as field of the derived public key

could we try and reuse that if possible?